units/suspend-then-hibernate.target

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Suspend; Idle into hibernate
+Description=Suspend; Hibernate if not used for a period of time.
```

Comment:
Please remove the periods from the end (Description is used as a phrase in sentences, so we'd have double punctuation.)

Reply:
OK, adjusted